products/BellHybrid/apps/application-bell-meditation-timer/models/ChimeVolume.cpp

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2017-2021, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#include "ChimeVolume.hpp"
+
+namespace app::meditation::models
+{
+
+    void ChimeVolume::setValue(std::uint32_t value)
+    {}
+    std::uint32_t ChimeVolume::getValue() const
+    {
+        return 5;
```

Comment:
should it return hard coded value?

Reply:
I added only a dummy implementation just to be able to compile. This should be properly implemented as part of BH-1433.

Reply:
Please add comment then.